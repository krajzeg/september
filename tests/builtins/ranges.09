print("This should print numbers from 1 to 5:")
for (i) in (Range(1,5)) {
	print(i)
}

print("This should print numbers from 1 to 4:")
openRange := Range(1,5,True)
for (i) in (openRange) {
	print(i)
}

