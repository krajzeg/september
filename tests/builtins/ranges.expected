This should print numbers from 1 to 5:
1
2
3
4
5
This should print numbers from 1 to 4:
1
2
3
4
